### PR TITLE
doc: user should not merge the pr too soon

### PR DESCRIPTION
**Pull request by yftacherzog**
We got user feedback saying it's unclear whether the PR created on their test repo should be merged or not once it's created. This change clarifies that.

closes: #154

### CODE REVIEW

**Review by ifireball — COMMENTED** [review]
it looks like a broken promise - "well tell you when..." (Never telling)

**Comment by yftacherzog** [comment]
> it looks like a broken promise - "well tell you when..." (Never telling)

We are telling -- in section `Trigger the Release`. Do you think we should mention that when we give the promise?

**Comment by ifireball** [comment]
> > it looks like a broken promise - "well tell you when..." (Never telling)
> 
> We are telling -- in section `Trigger the Release`. Do you think we should mention that when we give the promise?

I didn't I think about it, but yes I do. 'See in the "Trigger the Release" section' below sounds more reassuring. Alternatively we could maybe just link to the section.

**Comment by yftacherzog** [comment]
> > > it looks like a broken promise - "well tell you when..." (Never telling)
> > 
> > 
> > We are telling -- in section `Trigger the Release`. Do you think we should mention that when we give the promise?
> 
> I didn't I think about it, but yes I do. 'See in the "Trigger the Release" section' below sounds more reassuring. Alternatively we could maybe just link to the section.

ok. I wouldn't want them to click any link for that at this point, but I will mention it's there.

File: README.md [hunk 1]
```diff
@@ -263,7 +263,8 @@ to configure a Quay.io application and deploy `image-controller`.
 
 The UI should now display the Lifecycle diagram for your application. In the Components
 tab you should be able to see your component listed and you'll be prompted to merge the
-automatically-created Pull Request (don't do that just yet).
+automatically-created Pull Request (don't do that just yet. we'll have it merged in
+section [Trigger the Release](#trigger-the-release)).
```
Comment (ifireball) [inline README.md:267]: Perhaps say "Trigger and Release section" rather then "section Trigger and Release"

Comment (yftacherzog) [reply README.md:267]: I prefer it the way it is.

Comment (ifireball) [reply README.md:267]: ok.

**Review by ifireball — APPROVED** [review]

**Pull request merged**